emission/core/wrapper/suggestion_sys.py

```diff
@@ -403,7 +451,21 @@ def calculate_yelp_server_suggestion_nominatim(uuid):
         business_locations = {}
         begin_string_address, begin_address_dict = return_address_from_location_nominatim(start_lat, start_lon)
         end_string_address, end_address_dict = return_address_from_location_nominatim(end_lat, end_lon)
-        city = end_address_dict["city"]
+
+        try: 
+            city = end_address_dict["city"]
```

Comment:
just curious, why do you have two copies of this? Do you have two copies of the suggestionsys code? You should probably refactor that in a separate cleanup commit. Copy-and-paste is generally bad.

Reply:
There are two copies because both the single trip (for each trip) suggestion and the other suggestion function have the same problem when I try to index for the city. Should I make it into a function?